src/components/accession2/view/Accession2View.tsx

```diff
@@ -482,8 +482,19 @@ export default function Accession2View(props: Accession2ViewProps): JSX.Element
           <Box sx={editableParentProps}>
             <Icon name='notification' className={classes.iconStyle} />
             <Box sx={editableProps} onClick={() => setOpenEndDryingReminderModal(true)}>
-              <Typography paddingLeft={1}>
-                {accession?.dryingEndDate ? strings.END_DRYING_REMINDER_ON : strings.END_DRYING_REMINDER_OFF}
+              <Typography paddingLeft={1} lineHeight={isMobile ? 1.2 : 1.5}>
+                {accession?.dryingEndDate
+                  ? `${strings.END_DRYING_REMINDER} ${moment(accession.dryingEndDate).fromNow()}`
```

Comment:
this is cool! out of curiosity, does `moment` inherit its locale from global state somehow?